p11-kit/rpc-message.h

```diff
@@ -224,7 +225,7 @@ static const p11_rpc_call p11_rpc_calls[] = {
 	{ P11_RPC_CALL_C_GenerateKeyPair,      "C_GenerateKeyPair",      "uMaAaA",  "uu"                   },
 	{ P11_RPC_CALL_C_WrapKey,              "C_WrapKey",              "uMuufy",  "ay"                   },
 	{ P11_RPC_CALL_C_UnwrapKey,            "C_UnwrapKey",            "uMuayaA", "u"                    },
-	{ P11_RPC_CALL_C_DeriveKey,            "C_DeriveKey",            "uMuaA",   "u"                    },
+	{ P11_RPC_CALL_C_DeriveKey,            "C_DeriveKey",            "uMuaA",   "uPu"                    },
```

Comment:
This breaks compatibility at the protocol level; I would suggest bumping the protocol version and provide a proper fallback, as in b2dd8d5c73dab8e0f074b32d7bb393817964fb1d.